critiquebrainz/db/review.py

```diff
@@ -591,6 +591,16 @@ def get_popular(limit=None):
     shuffle(reviews)
     return reviews[:limit]
 
+def top_languages():
+    all_languages= []
+    for review in reviews:
```

Comment:
How do you get the variable `reviews`?

Reply:
Great if you could use `SQL` that I suggested for getting the `count` of the languages in reviews.

Reply:
isnt the variable reviews and all of it's attributes defined in the lines above?

Reply:
@IsaacZanoria you use the variable `reviews` but it doesn't look like it has been assigned a value inside the function at all, so Python will not be able to find it.